# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/generic/DocAccessible.h
# Commit: 09de143215a1
# Full Hash: 09de143215a13a22dcb2cd53d4ed1dfb60480c82
# Author: Atila Butkovits <abutkovits@mozilla.com>
# Date: 2025-09-30 02:48 +0300
# Regressor Bug: 1987945
# File Overlap Count: 2
# Overlapping Files: accessible/base/nsCoreUtils.cpp, accessible/tests/browser/relations/browser_anchor_positioning.js
# Description:
#   Revert "Bug 1987945: apply code formatting via Lando" for causing bustages at DocAccessible.cpp.
# ==============================================================================

diff -r 826eb4ac00dd -r 09de143215a1 accessible/generic/DocAccessible.h
--- a/accessible/generic/DocAccessible.h	Tue Sep 30 02:35:22 2025 +0300
+++ b/accessible/generic/DocAccessible.h	Tue Sep 30 02:48:50 2025 +0300
@@ -415,12 +415,6 @@
   void AttrElementWillChange(dom::Element* aElement, nsAtom* aAttr);
   void AttrElementChanged(dom::Element* aElement, nsAtom* aAttr);
 
-  /**
-   * Given an accessible, check if it is anchored to other frames, and
-   * refresh the cache on each of those frames' accessibles.
-   */
-  void RefreshAnchorRelationCacheForTarget(LocalAccessible* aTarget);
-
  protected:
   virtual ~DocAccessible();
 